client/src/components/styles/_mixin.scss

```diff
@@ -0,0 +1,19 @@
+@mixin mobile-break() {
+  @media screen and (max-width: 393px) {
```

Comment:
pls use breakpoints which are used everywhere and use variable for that

Reply:
Fixed